README.md

```diff
@@ -44,46 +42,321 @@ see https://github.com/ioBroker/ioBroker/wiki/Installation
+
+```
+  "log": {
+    ...
+    "transport": {
+      ...
+      "syslog1": {
+        "type": "syslog",
+        "enabled": false,
+        "host": "localhost",
+        "host_comment": "The host running syslogd, defaults to localhost.",
+        "port_comment": "The port on the host that syslog is running on, defaults to syslogd's default port(514/UDP).",
+        "protocol": "udp4",
+        "protocol_comment": "The network protocol to log over (e.g. tcp4, udp4, unix, unix-connect, etc).",
+        "path_comment": "The path to the syslog dgram socket (i.e. /dev/log or /var/run/syslog for OS X).",
+        "facility_comment": "Syslog facility to use (Default: local0).",
+        "localhost": "iobroker",
+        "localhost_comment": "Host to indicate that log messages are coming from (Default: localhost).",
+        "sysLogType_comment": "The type of the syslog protocol to use (Default: BSD).",
+        "app_name_comment": "The name of the application (Default: process.title).",
+        "eol_comment": "The end of line character to be added to the end of the message (Default: Message without modifications)."
+      }
+    }
+  },
+```
+
+### Controlling and Monitoring of Adapter processes
+
+#### Multiple Adapter instances
+**Feature Status: Stable**
+
+ioBroker allows to install multiple adapters on the sytem. For each adapter multiple instances can be created and started with an independent configuration.
+
+#### Start Adapter instances as normal processes
+**Feature Status: Stable**
+
+By default adapters are started by the js-controller as separate nodejs processes. They connect to the State and Objects DBs to read and write their data and some statistical information.
+
+The js-controller is monitoring the process-id of the started process and also controls stopping or restarting these processes.
+
+With this approach the whole iobroker system is very robust. A faulty adapter will only affect his own process and the js-controller and the other adapter processes are unaffected.
+
+The downside of the approach is that more RAM is needed because each node.js process needs 20-30MB RAM for the Node.JS part of it. With this a 1GB system is usually limited to run approx 10 to max 15 adapter processes. Beside using systems with more RAM also splitting adapters on multiplte hosts in a multihost environment could be an idea.
+
+#### Start Adapter instances in compact mode
+**Feature Status: Technology Preview since 2.0.0**
+
+The compact mode is developed especially for low memory systems. All adapters configured to run in compact mode will run inside the js-controller process.
+
+The positive effect of this is that each adapter needs 20-30MB less RAM because it do not need to have it's own node.js process handling. With this also a 512MB RAM system can run several adapter instances.
+
+The downside is that as soon as one adapter is reacting faulty and generate an unhandled error, the whole js-controller process will be restarted.
```

Comment:
Does this work automatically? If so, how?

Reply:
js-controller has an automatic restart ... a restart.js is executed which restarts the controller :-)